tests/h/cli/commands/authclient_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import mock
+import pytest
+
+from h import models
+from h.cli.commands import authclient as authclient_cli
+
+
+class TestAddCommand(object):
```

Comment:
Done. I've opted to make it another assertion in the same test as the simplest way to check that the result is "an object with properties X and Y with values foo and bar"